test/integration_test.js

```diff
@@ -779,10 +1059,10 @@ describe("Integration tests", () => {
             ]);
           });
 
-          it("should encode records when pushed to the server", () => {
```

Comment:
Ah, found it. https://github.com/Kinto/kinto.js/blob/sync-result-updated-list/test/integration_test.js#L1068